### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by octopuscaat** [BattlesnakeOfficial/feedback#172] [Q&A] [answered]
i am confused on why my code is not working and i dont know how to fix it. i use code like this: if(head.x == boardHeight-1) {
    move = "down"
  }
but my code only executes once and every time i try to use a loop it gives me an error. the api stuff and the starters are not helpful on this, and i have looked everywhere on the internet and can't find anything that helps me fix it. i have tried putting it inside the console.log('MOVE: ' + move)  response.status(200).send({move: move }) part, but it just gives me an error. how do i fix it?

**Comment by andrewdmaclean** [comment, marked as answer]
Hey @octopuscaat 👋 Sorry this got missed 😢 

If you are looking for real-time feedback be sure to head over to https://play.battlesnake.com/discord and post something in the #battlesnake-questions channel.
